src/main/java/nl/tudelft/scrumbledore/sprite/SpriteStore.java

```diff
@@ -126,7 +131,17 @@ private Sprite getSpriteFromFile(File file, String dir) {
     int pos = name.lastIndexOf('.');
     String id = name.substring(0, pos);
     String ext = name.substring(pos + 1);
-    return new Sprite(id, ext, dir);
+
+    Vector size = new Vector(0, 0);
```

Comment:
This one can be placed directly in return new Sprite(id, ext, dir, size);
<review